src/pages/en/guides/content-collections.mdx

```diff
@@ -376,6 +376,10 @@ const { Content } = await entry.render();
 
 This will generate a new page for every entry in the `blog` collection. For example, an entry at `src/content/blog/hello-world.md` will have a slug of `hello-world`, and therefore its final URL will be `/posts/hello-world/`.
 
+:::caution
```

Comment:
Instead of a scary caution, I'd suggest making the basic example use a `[...rest]` parameter as well. Less to worry about, and futureproofs your code for when subdirectories are added!